pkg/kfapp/ossm/ossm_installer.go

```diff
@@ -85,10 +86,21 @@ func (ossm *Ossm) Init(resources kftypesv3.ResourceEnum) error {
 		return internalError(err)
 	}
 
+	var dsErr error
+	if dsErr = ossm.migrateDSProjects(); dsErr != nil {
+		// Just log the error here but don't return
+		log.Error(dsErr, "Error while migrating DS Projects")
+	}
+
 	if err := ossm.processManifests(); err != nil {
 		return internalError(err)
 	}
 
+	// If there was an error with migrateDSProjects, return it here
+	if dsErr != nil {
+		return internalError(dsErr)
+	}
```

Comment:
At this point, we applied lots of other things so it might be tricky to handle this particular error or recover from it. I think it's good enough to just log all the errors related to ns changes above, when we call the `func`.

Reply:
Removed this in [25e3b2e](https://github.com/maistra/opendatahub-operator/pull/1/commits/25e3b2e821e23fb1b789205201ccefe5a0160732)